cloudinit/config/cc_set_passwords.py

```diff
@@ -112,8 +112,13 @@ def handle_ssh_pwauth(pw_auth, distro):
         LOG.debug("No need to restart SSH service, %s not updated.", cfg_name)
         return
 
-    distro.manage_service("restart", distro.get_option("ssh_svcname", "ssh"))
-    LOG.debug("Restarted the SSH daemon.")
+    service = distro.get_option("ssh_svcname", "ssh")
+    try:
+        distro.manage_service("restart", service)
+    except subp.ProcessExecutionError as e:
+        LOG.warning("Failed to restart the SSH deamon. %s: %s", service, e)
```

Comment:
Generally let's try to avoid the words "Failed" and "error" in warning level logs as they are confusing when you grep in /var/log/cloud-init.log. If we stick with this approach, please replace Failed 
```suggestion
        LOG.warning(f"Wrote '{cfg_name} {cfg_val}' to {DEF_SSHD_CFG}. Unable to restart the SSH deamon {service}: {e}")
```

Reply:
Thanks for the suggestion @blackboxsw.
 
> I wonder if we might be able to add a "status" command to `distro.manage_service` to report whether or not the service actually exists. We could then try/except around a call to `distro.manage_service("status", service)`.

I was thinking exactly about that. Let me rework this PR.